test.py

```diff
@@ -0,0 +1,15 @@
+import requests
```

Comment:
First, you shouldn't push your API key. I'd recommend that you delete this one and get a new key.
Second, the tests are to be written in [`tests/test_observables.py`](https://github.com/intelowlproject/IntelOwl/blob/master/tests/test_observables.py#L63). You can see how it's done for other analyzers and follow the same pattern. Make one entry for it under `IPAnalyzersTests` class and same entry in `DomainAnalyzersTest` class.
Third, you can delete this file.

Reply:
I will delete the test file. The purpose of this file is api test to see if it works okay, it does not involve or affect other files.

Reply:
Yes, I understood. However, we do write test cases for all analyzers like I mentioned. So instead of having a individual test file, these should be under tests/test_observables.py following the same pattern as others.

Reply:
What files will I have to edit in the test folder? Besides the file tests / test_observables.py, are there any other files to edit?

Reply:
Nope, only that one.